learning/features/assetlib/uploading_to_assetlib.rst

```diff
@@ -0,0 +1,105 @@
+.. _doc_uploading_to_assetlib:
+
+Submitting to the Asset Library
+===============================
+
+Introduction
+------------
+
+This tutorial aims to serve as both a guide on how you can submit your own assets
+to the Godot Asset Library and share them with the Godot community.
+
+As mentioned in the :ref:`doc_using_assetlib` document, in order to be able to
+submit assets to the AssetLib, you need to have a registered account, and be
+logged in.
+
+Submitting
+----------
+
+Once you are logged in, you will be able to head over to the "Submit Assets" page
+of the AssetLib, which will look like this:
+
+|image0|
+
+While it may look like a lot (and there is more to scroll down), each field is
+described in terms of what you should put in. We will nonetheless go over what
+is required in the submission form here as well.
+
+* **Asset Name**: 
+    The name of your asset. Should be a unique, descriptive title of 
+    what your asset is.
+* **Category**: 
+    The category that your asset belongs to, and will be shown in
+    search results. The category is split into **Addons** and **Projects**. Assets
+    of the Project type (Templates, Demos, Projects) do not show up in the in-editor
+    AssetLib, and can only be downloaded from the website version of the AssetLib,
+    as well as from the provider directly (see below).
```

Comment:
Assets from the project type are downloadable from the project manager (Assetlib tab). The web frontend is not required to browse the assetlib in any way.